contrast-rest-dotnet/Model/Server.cs

```diff
@@ -113,24 +114,24 @@ public class Server
         /// <summary>
         /// Server environment. Allowed values: DEVELOPMENT, QA, PRODUCTION.
         /// </summary>
-        [DataMember(Name = "environment")]
+        [JsonProperty(PropertyName = "environment")]
         public string Environment { get; set; }
 
         /// <summary>
         /// Gets the hostname of this server.
         /// </summary>
-        [DataMember(Name = "hostname")]
+        [JsonProperty(PropertyName = "hostname")]
         public string Hostname { get; set; }
 
-        [DataMember(Name = "lastActivity")]
+        [JsonProperty(PropertyName = "lastActivity")]
         private long LastActivityRawValue { get; set; }
 
         /// <summary>
         /// Gets the last time any activity was received from this server.
         /// </summary>
         public DateTime? LastActivity { get; set; }
```

Comment:
This field doesn't do anything? You can use DateTime fields directly with the resolver I sent

Reply:
Done. Those kind of fields were removed and used JsonConverter with DateTime fields.